apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/layout.tsx

```diff
@@ -13,6 +13,7 @@ import {
 } from '@fxa/payments/ui/server';
 import { getFakeCartData } from '../../../../../_lib/apiClient';
```

Comment:
Suggestion: rename to `apps/payments/next/app/_lib/apiClient` similar to how it's written for `apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/page.tsx` L11.

Reply:
Good catch!
Looks like this _might_ be the only one:

```sh
git grep -Pin "\/\.\.\/_lib\/" apps libs | cat

apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/layout.tsx:14:import { getFakeCartData } from '../../../../../_lib/apiClient';
```

versus

```sh
git grep -Pin "\/\w+\/_lib\/" apps libs | cat

apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx:16:} from 'apps/payments/next/app/_lib/apiClient';
apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/page.tsx:11:import { getFakeCartData } from 'apps/payments/next/app/_lib/apiClient';
```